test/UnbrandedProjects/Platform-OpenAI-TypeSpec/src/Generated/Internal/MultipartFormDataBinaryContent.cs

```diff
@@ -0,0 +1,197 @@
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ClientModel;
+using System.Globalization;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace OpenAI
+{
+    internal class MultipartFormDataBinaryContent : BinaryContent
+    {
+        private readonly MultipartFormDataContent _multipartContent;
+        private static readonly Random _random = new Random();
+        private static readonly char[] _boundaryValues = "0123456789=ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz".ToCharArray();
+
+        public MultipartFormDataBinaryContent()
+        {
+            _multipartContent = new MultipartFormDataContent(CreateBoundary());
+        }
+
+        public string ContentType
+        {
+            get
+            {
+                return _multipartContent.Headers.ContentType.ToString();
+            }
+        }
+
+        internal HttpContent HttpContent => _multipartContent;
+
+        private static string CreateBoundary()
+        {
+            Span<char> chars = new char[70];
+            byte[] random = new byte[70];
+            _random.NextBytes(random);
+            int mask = 255 >> 2;
+            for (int i = 0; i < 70; i++)
```

Comment:
There should be a `Debug.Assert(_boundaryValues.Length - 1 == mask);` preceeding this line.